src/engine/experimental/blob-extension.md

```diff
@@ -44,7 +44,8 @@ This structure has the syntax of `ExecutionPayloadV2` and appends a single field
 - `blockHash`: `DATA`, 32 Bytes
 - `transactions`: `Array of DATA` - Array of transaction objects, each object is a byte list (`DATA`) representing `TransactionType || TransactionPayload` or `LegacyTransaction` as defined in [EIP-2718](https://eips.ethereum.org/EIPS/eip-2718)
 - `withdrawals`: `Array of WithdrawalV1` - Array of withdrawals, each object is an `OBJECT` containing the fields of a `WithdrawalV1` structure.
-- `excessDataGas`: `QUANTITY`, 256 bits
+- `excessDataGas`: `QUANTITY`, 64 Bits
+- `dataGasUsed`: `QUANTITY`, 256 bits
```

Comment:
order should be reversed, edg then dgu (assuming PR is merged as-is): https://github.com/ethereum/EIPs/pull/7062/files

Reply:
> edg then dgu

i like this notation, we should consider introducing it for all Engine API fields and definitions. This would make all other concerns fade away

Reply:
I think we should reverse the order and then can merge this PR